svc-managers/mgrmodel/manager.go

```diff
@@ -123,6 +123,7 @@ type RAManager struct {
 	State           string     `json:"State"`
 	Description     string     `json:"Description"`
 	LogServices     *dmtf.Link `json:"LogServices"`
+	Health          string     `json:"Health"`
```

Comment:
is health a redfish property?

Reply:
this struct is for saving odimra details to DB, hence added it here